### ISSUE TITLE
No need for aggregatename when resizing a volume

### ISSUE THREAD

**Issue by ansibleguy76** [ansible-collections/netapp.ontap#119]
### Summary

Why do you need the aggregate name when you want to resize a volume.

a volumes name is already identified by name, vserver & cluster.

### Component Name

na_ontap_volume

### Additional Information

```yaml (paste below)
- name: Resize vol0
  na_ontap_volume:
    state: present
    name: vol0
    vserver: "{{ nodes[0].name }}"
    aggregate_name: "aggr0_{{ nodes[0].name | replace('-','_') }}"
    size: 13
    size_unit: gb
    percent_snapshot_space: 0
    space_guarantee: none
    <<: *auth
```

**Comment by lonico** [comment]
Agreed.  The example is not intended to show a minimal set of options, as unchanged values are ignored.
But we'll remove the aggregate as it implies a volume move.
DEVOPS-5665

**Comment by lonico** [comment]
I verified that you do not need an aggregate name to resize a volume when using a recent version of Ansible.

And the documentation does not show such an example.

Did you observe the issue with an older collection?

**Comment by lonico** [comment]
Oh, I understand the issue now.

Since REST does not show node or cluster resources, when we query for vol0, we don't find anything.  And then we try to create it.  In order to create, we need an aggregate.

**Comment by carchi8py** [comment]
This was fixed in 22.1.0 which was released last week.
